src/components/user-card/UserCard.js

```diff
@@ -1,9 +1,31 @@
 import React from 'react';
 
 //todo здесь нужно сделать импорт стилей из './UserCard.scss'
+import './UserCard.scss'
+import {usersList} from "../../constants";
 
 
 // todo здесь нужно сделать экспорт функии под названием UserCard с аргументом props
+export const UserCard = (props) =>{
+   const user=usersList[0];
```

Comment:
A little bit incorrect using of user
You need to pass `user ` to UserCard in place where you render UserCard as tag (< UserCard/>). I suppose, in App.js
and here you need to get user from `props` using destructuring

Reply:
Please go to App.js and try to render your UserCard